README.md

```diff
@@ -414,7 +416,7 @@ Based on [FHIR Observation](https://hl7.org/fhir/R4B/observation.html), the foll
 |component[x]>value|optional [Quantity](https://hl7.org/fhir/R4B/datatypes.html#Quantity)|e.g. `{"code":"mm[Hg]","system":"http://unitsofmeasure.org","unit":"mmHg","value":120}`|Use one of the units listed below.|
 |effective|[Period](https://hl7.org/fhir/R4B/datatypes.html#period), [Instant](https://hl7.org/fhir/R4B/datatypes.html#instant) or [DateTime](https://hl7.org/fhir/R4B/datatypes.html#dateTime)|-|Use DateTime/Instant for instant observation and periods for longer observations (e.g. more than 1 minute).|
 
-#### Blood Pressure
+#### /users/$userId$/observations/bloodPressure/$observationId$
```

Comment:
Firebase paths always need to follow a collection/document/collection order. I would therefore suggest to drop the observations path element.

Applies to the other paths as well.

Reply:
okay, could we then just add a observation-suffix, maybe? i.e. bloodPressureObservations?